workshop/exercise-3/README.md

```diff
@@ -1 +1,87 @@
 # Exercise 3 - Deploy Guestbook with Istio Proxy 
+
+The Guestbook application consists of a web front end, Redis master for storage, and replicated set of Redis slaves. We will deploy that application on Kubernetes with Istio manual injection.
+
+**Make sure you are under the guestbook-go application folder**
+
+## Create Redis backend
+The Redis backend provides the persistance service to the application. It consists of the master and slave modules. We will create the controllers and service for both master and slave.
+  ``` sh
+  kubectl create -f redis-master-controller.json
+  kubectl create -f redis-master-service.json
+  kubectl create -f redis-slave-controller.json
+  kubectl create -f redis-slave-service.json
```

Comment:
it would be good to switch to deployment instead:  https://github.com/IBM/guestbook/blob/master/v2/redis-master-deployment.yaml